core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -1350,6 +1443,24 @@ private Class<?> getContainerType(Type parameterizedType) {
             return null;
         }
 
+        @Nullable
+        private Class<?> getRawContainerType(Type parameterizedType) {
```

Comment:
```suggestion
        private static Class<?> getRawContainerType(Type parameterizedType) {
```

Reply:
I use `rawType` instead of `rawContainerType()`.
Thus, i removed method `rawContainerType()` and field!

Reply:
Oops, i forgot it. 
Now, i removed it!